libs/logging/src/log_event_ids.ts

```diff
@@ -1,36 +1,34 @@
+/**
+ * Do not edit this file directly! It's generated and manual changes will be overwritten.
+ * To add a log event, add it to /config/log_event_details.toml and run `pnpm generate-types`.
+ */
+
 import { throwIllegalValue } from '@votingworks/basics';
 import { LogEventType } from './log_event_types';
 import { LogSource } from './log_source';
 
-/**
- * In order to add a new log event you must:
- * 1. Add the new event to the enum LogEventId
- * 2. Define a LogDetails object representing the information about that log event.
- * 3. Add a case statement to the switch in getDetailsForEventId returning your new LogDetails object when you see your new LogEventId.
- * You will then be ready to use the log event from your code!
- */
-
+export interface LogDetails {
+  eventId: LogEventId;
+  eventType: LogEventType;
+  defaultMessage?: string;
+  documentationMessage: string;
+  // Only includes the log in the documentation file for the given apps, if not specified will be included in all apps documentation.
+  restrictInDocumentationToApps?: LogSource[];
+}
 export enum LogEventId {
-  // Election configuration logs
   ElectionConfigured = 'election-configured',
   ElectionUnconfigured = 'election-unconfigured',
-
-  // System level logs
```

Comment:
I'll manually sort and add these comments back the `.toml` file once we land on format